client/src/components/Header.js

```diff
@@ -16,26 +16,20 @@ const HeaderContainer = styled.header`
 const MenuButton = styled.button`
   background: none;
   border: none;
-  justify-content: flex-start;
 `;
 const Img = styled.img`
```

Comment:
you wont need this one, just add the style to your `<MenuButton>` like this 

```JSX
<MenuButton>
...
img{
...
}
</MenuButton>

```

Reply:
True! Thanks! Keep forgetting about that, since it still looks "new" to me! Will try to implement it more often.